src/components/AllArticles.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import { getArticles } from "../../api";
+import Header from "./Header";
+import Footer from "./Footer";
+import { Link } from "react-router-dom";
+import { CarouselItem } from "react-bootstrap";
+import { Carousel } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+export default function AllArticles() {
+  const [articles, setArticles] = useState([]);
```

Comment:
Great that you've announced this state here, as opposed to in the App
It means you're keep state as low down the component tree as possible, which is great 🙌